### PR TITLE
Command line argument of learning rate scheduler

### PR DESCRIPTION
**Pull request by hlh981029**
#120

### CODE REVIEW

**Comment by hlh981029** [comment]
https://github.com/alibaba/EasyNLP/blob/169bf0a706985a06fc2598960fdafc8c3be64944/easynlp/core/trainer.py#L325-L334

这里的逻辑我不太理解，比如，`self._optimizer` 是 `BertAdam` 的时候 `self._lr_scheduler` 为 `None`，这里是需要调用 `self._optimizer.step()` 的。
